ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -51,20 +51,39 @@ $services.localization.render('contrib.meeting.home.presentation', ['[[**', '**&
   'leader': {"type":"text","size":10,"html":true},
   'place': {"type":"text","size":10},
   'startDate': {"type":"text","size":10},
+  'endDate': {"type":"hidden"},
   'durationDisplay': {"type":"text","filterable":true,"sortable":false,"html":true},
   'status': {"type":"list","size":10},
   '_actions': {"sortable":false,"filterable":false,"html":true,"actions":["delete", "edit"]}
 })
-#set ($options = {
-  'className': 'Meeting.Code.MeetingClass',
-  'translationPrefix': 'meeting.livetable.',
-  'tagCloud': true,
-  'rowCount': 15,
-  'maxPages': 10,
-  'selectedColumn': 'startDate',
-  'defaultOrder': 'desc'
-})
-#set ($columns = ['doc.title', 'leader', 'place', 'startDate', 'durationDisplay', 'status', '_actions'])
+#set ($currentDate = $xwiki.jodatime.dateTime.millis)
+#set ($endDate = $xwiki.jodatime.dateTime.plusYears(1).millis)
+#set ($parameter= $request.getParameter("meeting"))
+#if($parameter=="all")
+  $services.localization.render('contrib.meeting.home.allMeetingsMessage',['[[', "&gt;&gt;${doc}]]"])
+  #set ($options = {
+    'className': 'Meeting.Code.MeetingClass',
+    'translationPrefix': 'meeting.livetable.',
+    'tagCloud': true,
+    'rowCount': 15,
+    'maxPages': 10,
+    'selectedColumn': 'startDate',
+    'defaultOrder': 'desc'
+  })
+#else
+  $services.localization.render('contrib.meeting.home.initialMessage',['[[**','&gt;&gt;||queryString="meeting=all"]]'])
+  #set ($options = {
+    'className': 'Meeting.Code.MeetingClass',
+    'translationPrefix': 'meeting.livetable.',
+    'tagCloud': true,
+    'rowCount': 15,
+    'maxPages': 10,
+    'selectedColumn': 'startDate',
+    'extraParams':"&amp;endDate=${currentDate}-${endDate}",
```

Comment:
A space would be needed after `:`, like you did for the others

Reply:
I see that for both declarations of `$options` there are some properties that remain the same, and only a few change depending on `$parameter`. Maybe you could change only some properties inside your `if` to not duplicate code